src/particles/transformation/CoordinateTransformation.H

```diff
@@ -21,14 +21,14 @@ namespace transformation
      */
     enum class Direction
     {
-        T2Z,
-        Z2T
+        To_Fixed_S,
+        To_Fixed_T
```

Comment:
Just a bit more in our style:
```suggestion
        to_fixed_s,
        to_fixed_t
```